src/analyze/analyze.c

```diff
@@ -66,7 +68,9 @@
 static enum dot {
         DEP_ALL,
         DEP_ORDER,
-        DEP_REQUIRE
+        DEP_REQUIRE,
+        DEP_FAILURE,
+        DEP_TRIGGERS
```

Comment:
nitpick: doesn't matter at all, but for most enums we these days tend to put a final "," after the last value, to minimize diffs for later changes...

Reply:
done